src/IdentityServer/IdentityServerTools.cs

```diff
@@ -48,22 +48,23 @@ public IdentityServerTools(IServiceProvider serviceProvider, IIssuerNameService
     /// <exception cref="System.ArgumentNullException">claims</exception>
     public virtual async Task<string> IssueJwtAsync(int lifetime, IEnumerable<Claim> claims)
     {
+        var tokenType = OidcConstants.TokenTypes.AccessToken;
         var issuer = await IssuerNameService.GetCurrentAsync();
-        return await IssueJwtAsync(lifetime, issuer, claims);
+        return await IssueJwtAsync(lifetime, issuer, tokenType, claims);
     }
 
     /// <summary>
     /// Issues a JWT.
     /// </summary>
     /// <param name="lifetime">The lifetime.</param>
-    /// <param name="issuer">The issuer.</param>
+    /// <param name="tokenType">The token type.</param>
     /// <param name="claims">The claims.</param>
     /// <returns></returns>
     /// <exception cref="System.ArgumentNullException">claims</exception>
-    public virtual Task<string> IssueJwtAsync(int lifetime, string issuer, IEnumerable<Claim> claims)
+    public virtual async Task<string> IssueJwtAsync(int lifetime, string tokenType, IEnumerable<Claim> claims)
```

Comment:
Oh wait, this is a breaking change. Need to fix the caller instead I think.

Reply:
To fix the caller, we have to add a dependency on the `IssuerNameService` from the `DefaultBackChannelLogoutService`, which technically is also a breaking change. Maybe we don't need this in a 6.3 patch release?

Reply:
If it's a bug and not working as expected, then I don't think it's a problem to make a breaking change.

Reply:
Okay, then I'll keep this for the 6.3 branch, but still rework the fix slightly so that we're not changing semantics of the method in IdentityServerTools. This way the "breaking" change only affects the bug, and won't surprise people (the other change would probably have compiled without an error!)